src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -791,8 +791,21 @@ x3dom.registerNodeType(
                         hasColor = false;
                     }
                     this._mesh._numColComponents = numColComponents;
+                    if (fieldName.startsWith("attrib")) {
+                        var attrName = fieldName.slice(7);
```

Comment:
The name is given during the call of FiledChanged on the FloatVertexAttribute element
then the FloatVertexAttribute call the fieldChanged from IndexedFaceSet (the code above) with attrib_*name*
see : 16fefd1
```javascript
fieldChanged: function (fieldName) {
                 var that = this;
                 if (fieldName == "value" ||  fieldName == "numComponents") {
                     Array.forEach(this._parentNodes, function (node) {
                         node.fieldChanged("attrib_"+that._vf.name);
                     });
                 }
             }
```